### PR TITLE
Security screen

### PR DESCRIPTION
**Pull request by zachgibb**

### CODE REVIEW

File: src/screens/Settings/Security/Security.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+import { t } from 'translations/i18n';
+
+export default function Security(props) {
+  return (
+    <Layout preload={false}>
+      <Body scrollable style={styles.content}>
+        <Link title={t('settings.seed_words')} to="SeedWords" />
+        {props.isSignedIn && (
+          <Link title="Change Password" to="ChangePassword" />
```
Comment (ericgibb) [inline src/screens/Settings/Security/Security.js]: language update to `{t('settings.change_password_title')}` or something

File: src/screens/Settings/Settings/Settings.js [hunk 2]
```diff
@@ -30,12 +29,9 @@ export default function Settings(props) {
...
           }
         />
-        {props.isSignedIn && (
-          <Link
-            title={t('profile.title')}
-            to="Profile"
-          />
-        )}
+
+        <Link title="Security" to="Security" />
```
Comment (ericgibb) [inline src/screens/Settings/Settings/Settings.js]: i18n the title here?

File: src/screens/Settings/index.js [hunk 3]
```diff
@@ -37,7 +39,26 @@ const RoutingStack = createStackNavigator(
...
+          rightAction: null,
+        }),
+    },
+    Security: {
+      screen: Security,
+      navigationOptions: navProps =>
+        getNavigationOptions({
+          ...navProps,
+          leftAction: 'back',
+          title: 'Security',
```
Comment (ericgibb) [inline src/screens/Settings/index.js]: i18n

File: src/screens/Settings/Security/Security.js (hunk 1 above)
Comment (zachgibb) [reply src/screens/Settings/Security/Security.js]: fixed!

File: src/screens/Settings/Settings/Settings.js (hunk 2 above)
Comment (zachgibb) [reply src/screens/Settings/Settings/Settings.js]: fixed!

File: src/screens/Settings/index.js (hunk 3 above)
Comment (zachgibb) [reply src/screens/Settings/index.js]: fixed!

**Pull request merged**